packages/data-stores/src/queries/use-site-intent.ts

```diff
@@ -1,18 +0,0 @@
-import { useQuery } from '@tanstack/react-query';
```

Comment:
I couldn't find any usages of this hook hence the removal.

Reply:
There are a few comments throughout the codebase that will no longer make sense and we might need to update them:

https://github.com/Automattic/wp-calypso/blob/a637ef7503821ee8bcaf6972e3dd41febca2c136/apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/lib/site-intent/use-site-intent.js#L1

https://github.com/Automattic/wp-calypso/blob/a637ef7503821ee8bcaf6972e3dd41febca2c136/apps/wpcom-block-editor/src/wpcom/features/use-site-intent.js#L1

Reply:
Mmh, it may be worthwhile exploring if that's still something we want to do. Looping in @okmttdhr & @arthur791004 as they discussed this in https://github.com/Automattic/wp-calypso/pull/73565#discussion_r1113839120 – what do you think? We removed the `useSiteIntent` from the `data-stores` package as it doesn't seem to be used anywhere. We still think the `data-stores` package isn't the right fit for the hook regardless.

Reply:
Seems like usages of `useSitenIntent` were introduced after I opened this PR. I reverted the commit which removed it with https://github.com/Automattic/wp-calypso/pull/78682/commits/c43cb08749d1659c039e338ed48d8dec83bb1757 but we still should find a better place for it.